### PR TITLE
Social Links component

### PR DESCRIPTION
**Pull request by RisingGeek**
Component to display social/technical profile URLs.
image

### CODE REVIEW

**Comment by ManojSatishkumar** [comment]
Can you add social icons and call this component as "social links" ?

**Comment by RisingGeek** [comment]
> Can you add social icons and call this component as "social links" ?

Done

File: src/modules/resume/social-links/variants/basic/properties.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import propertyTypes from 'msk-property-types';
+import themes from 'msk-themes';
+import mskPropertyTypes from 'msk-property-types';
```
Comment (ManojSatishkumar) [inline src/modules/resume/social-links/variants/basic/properties.js:3]: Duplicate import

Comment (RisingGeek) [reply src/modules/resume/social-links/variants/basic/properties.js:2]: Removed duplicate imports

**Pull request merged**